conftest.py

```diff
@@ -379,7 +379,7 @@ def t3k_device_mesh(request, silicon_arch_name, silicon_arch_wormhole_b0):
         num_devices_requested = min(request.param, len(device_ids))
     except (ValueError, AttributeError):
         num_devices_requested = len(device_ids)
-
+    num_devices_requested = 2
```

Comment:
?

Reply:
My bad, removing this and running multichip.